docs/release-notes/index.md

```diff
@@ -25,11 +25,9 @@ To check for security updates, go to [Security announcements for the Elastic sta
 
 ### Features and enhancements [logstash-9.2.0-features-enhancements]
 
-#### Highlights
+#### Persistent queue (PQ} compression [logstash-9.2.0-pq-compression]
 
-##### Logstash 9.2 introduces PQ compression
```

Comment:
Three heading levels with no content inbetween can signal a structural issue.  I restructured the content to [hopefully] achieve the emphasis you were going for, @robbavey, but eliminate a heading level. LMKWYT

Reply:
Much better!